src/docs/user.yaml

```diff
@@ -12,7 +12,7 @@ paths:
           description: 'ID of the user'
           examples:
             'real ID of an available user':
-              value: '62eb038c20b4f5b0bd8c7676'
+              value: '62e7aa14c0f7ad46403b142e'
```

Comment:
Did you delete your user @okurezgi? It appears as a deleted user. Its name says "ID of _available user_" lol

Reply:
if it was an id from facebook one, i may have deleted it for trying it again. It may change the id. But you fixed the issue i guess, right?

Reply:
Oh I see. Yes, fixed it. Now it shows a user when is used.